IS0402Test.py

```diff
@@ -843,7 +847,7 @@ def test_21_7(self, test):
                                   expected_ids=ids,
                                   expected_since="0:0", expected_until=ts[-1], expected_limit=count)
 
-        resources = response[1].json() # valid json guaranteed by check_paged_response
+        resources = response[1].json()  # valid json guaranteed by check_paged_response
```

Comment:
Why's that space necessary?

Reply:
It's enforcing the rule from https://www.python.org/dev/peps/pep-0008/#inline-comments, second sentence.

Reply:
OK. Thanks. (I probably should have just put that comment on the previous line.)